grpc/interceptor/logger_test.go

```diff
@@ -18,9 +18,9 @@ import (
 // LogLine is only used to parse log lines from interceptors making it
 // easier to test
 type logLine struct {
-	Msg       string `json:"msg"`
-	RequestID string `json:"request_id"`
-	Duration  string `json:"duration"`
+	Msg       string      `json:"msg"`
+	RequestID string      `json:"request_id"`
+	Duration  json.Number `json:"duration,Number"`
```

Comment:
look below it at https://github.com/namely/mjolnir/pull/11/files#diff-a4ec4a083b9c6bc6004ca2e26846896fR60 that checks duration was included in the "finished rpc" log